New module feature enable capability (#8)
mzagozen/notconf: README.md

container. Therefore the quickest way to use custom modules is to bind mount a
host directory to the `/yang-modules` directory in the container.

Yang modules could also contain optional features that, by default, are disabled. 

> At startup the container will look for the file `/yang-modules/enable-feature.csv`
> to determine if some features should be enabled. Each row of the file should start
> with a module name followed by comma and the name of a feature to enable.
> Note that the last line of the file MUST terminate with a newline.

Reviewer: Change file name to `enable-features.csv` and clarify row format:

```suggestion
At startup the container will look for the file `/yang-modules/enable-features.csv`
to determine if some features should be enabled. Each row of the file should start
with a module name followed by comma and the name of a feature to enable,
one pair per line.
```

``` shell
# build the custom notconf-test image

> ❯ docker build -f Dockerfile.yang -t notconf-dzs --build-arg COMPOSE_PATH=dzs/yang-modules .

Author: Right, "dzs" instead of "test" it is an oversight. 
The folder name `yang-modules` however is necessary so I corrected like this:

`❯ docker build -f Dockerfile.yang -t notconf-test --build-arg COMPOSE_PATH=test/yang-modules .`
